Tools/python/MetaSubmitter.py

```diff
@@ -267,14 +269,18 @@ def set_splitting(samples, dataset, jobtype='default', data_json=None, default_f
 
     if limit_ttbar:
         d = { # get ~400/fb
-            'ttbarht0600_2017':  (20, 726800),
-            'ttbarht0800_2017':  ( 8, 300800),
-            'ttbarht1200_2017':  ( 2,  52500),
-            'ttbarht2500_2017':  ( 1,   1000),
+            #'ttbarht0600_2017':  (20, 726800),
+            #'ttbarht0800_2017':  ( 8, 300800),
+            #'ttbarht1200_2017':  ( 2,  52500),
+            #'ttbarht2500_2017':  ( 1,   1000),
             'ttbarht0600_2018':  (25, 726800),
             'ttbarht0800_2018':  (11, 300800),
             'ttbarht1200_2018':  ( 2,  52500),
             'ttbarht2500_2018':  ( 1,   1000),
+            'ttbarht0600_2017':  (200, 7268000),
+            'ttbarht0800_2017':  ( 80, 3008000),
+            'ttbarht1200_2017':  ( 20,  525000),
+            'ttbarht2500_2017':  ( 10,   10000),
             }
```

Comment:
Not sure if this is intentional? It looks like this is to get an effective luminosity of closer to 4000 fb^-1 worth of ttbar events? We can always turn off the limit_ttbar option when needing more stats.

Reply:
I think it was intentional because I was trying to get more stats for ML at some time without realizing the limit_ttbar option...